contracts/test/ProtocolAdapter.t.sol

```diff
@@ -1,36 +1,125 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.30;
 
-import {Pausable} from "@openzeppelin-contracts/utils/Pausable.sol";
+import { Pausable } from "@openzeppelin-contracts/utils/Pausable.sol";
```

Comment:
Please use `forge fmt`. If you use vscode, it will automatically use forge as the formatter for `.sol` files (see https://github.com/anoma/evm-protocol-adapter/blob/main/.vscode/settings.json)